Gemfile

```diff
@@ -1,10 +1,11 @@
+# frozen_string_literal: true
 source "https://rubygems.org"
 
 gemspec
 
 # NOTE: do not submit PRs to add pry as a dep, add to your Gemfile.local
 group :development do
-  gem "chefstyle", git: "https://github.com/chef/chefstyle.git", branch: "master"
+  gem "chefstyle", "= 1.1.3"
```

Comment:
Using chefstyle from git just resulted in constant errors locally and the inability to use dependabot to bump style on our own terms. This way it'll open PRs to bump the version and we can evaluate any fixes that need to happen instead of breaking community PRs potentially due to unrelated rubocop changes.